src/extractor/scripting_environment_lua.cpp

```diff
@@ -945,10 +945,6 @@ Sol2ScriptingEnvironment::GetStringListFromTable(const std::string &table_name)
     auto &context = GetSol2Context();
     BOOST_ASSERT(context.state.lua_state() != nullptr);
     std::vector<std::string> strings;
-    if (!context.profile_table[table_name])
```

Comment:
otherwise we have crash in runtime on some assertion inside sol2 that it expected boolean, but got table, i.e. it probably infers expected type from context(in our case `if` condition) and assert if real type corresponding it. As understand here our goal was to check that we really have this table in script, but this case is fully covered by condition a couple of lines below(`if (table.valid())`).